Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1878,6 +1878,11 @@ - (CGPoint)_scrollView:(UIScrollView *)scrollView adjustedOffsetForOffset:(CGPoi
 #if HAVE(UISCROLLVIEW_ASYNCHRONOUS_SCROLL_EVENT_HANDLING)
 - (void)_scrollView:(UIScrollView *)scrollView asynchronouslyHandleScrollEvent:(UIScrollEvent *)scrollEvent completion:(void (^)(BOOL handled))completion
 {
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+    if ([scrollEvent _scrollDeviceCategory] == _UIScrollDeviceCategoryOverlayScroll)
+        return;
```

Comment:
You need to call the completion handler with "true" ("I handled it, don't worry about it"), or the events are going to stack up on UIScrollView.